.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: Test
+
+on:
+  push:
+    branches: [ core ]
+  pull_request:
+    branches: [ core ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+          - 2.3.8
+          - 2.4.5
+          - 2.5.3
+          - 2.6.3
```

Comment:
shall we take the opportunity to remove 2.3, and 2.4, add 2.7 and 3.0 ?
separate PR is better?